addons/mail/static/src/components/thread_view/thread_view.xml

```diff
@@ -38,7 +38,7 @@
                         isDoFocus="props.isDoFocus"
                         showAttachmentsExtensions="props.showComposerAttachmentsExtensions"
                         showAttachmentsFilenames="props.showComposerAttachmentsFilenames"
-                        textInputSendShortcuts="(threadView.thread.model === 'mail.channel' and threadView.thread.mass_mailing) ? ['ctrl-enter', 'meta-enter'] : ['enter']"
+                        textInputSendShortcuts="(threadView.thread.model === 'mail.channel' and (threadView.thread.mass_mailing or (threadView.thread.chatWindow and env.messaging.device.isMobile))) ? ['ctrl-enter', 'meta-enter'] : ['enter']"
```

Comment:
It's not good to reference `chatWindow` (parent) in `threadView` (child).

If we do need this condition to resolve the issue, the value will need to come from the parent by a prop, or be a (computed) field on the thread view(er) model.